libs/utils/state_wrapper.go

```diff
@@ -0,0 +1,34 @@
+package utils
```

Comment:
* `libs/utils` does not make sense(even though I made it, but I don't like this) 
* If this is needed for service management, we should make a `service` pkg.

Reply:
done